packages/client-sdk-nodejs/src/config/middleware/experimental-metrics-logging-middleware.ts

```diff
@@ -105,6 +105,8 @@ export class ExperimentalMetricsLoggingMiddleware extends ExperimentalMetricsMid
   private static elu: EventLoopUtilization;
   private static isLoggingStarted = false;
   static numActiveRequests = 0;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  private static intervalId: any | null = null; // Store the interval ID
```

Comment:
looking at this code, i'm not totally sure why all of these variables are static, but i guess we don't need to deal with that now.

Reply:
I am going to move this to another middleware so will address it for sure on that PR!